src/pal/src/misc/sysinfo.cpp

```diff
@@ -86,6 +87,9 @@ SET_DEFAULT_DEBUG_CHANNEL(MISC);
 #endif
 #endif // __APPLE__
 
+//masks for PAL_HasFunctionality
+#define GETCPU 0x0001
```

Comment:
Please move this into pal.h since the coreclr runtime needs to have access to it.
Please also rename it to something like PAL_FUNCTIONALITY_GET_CURRENT_CPU or PAL_CAPABILITY_GET_CURRENT_CPU.